src/modules/document-caching/AmpDocumentNetworkFirst.ts

```diff
@@ -18,25 +18,30 @@
 import { NetworkFirst } from 'workbox-strategies';
 import { AMP_PUBLISHER_CACHE } from './constants';
 
+// TODO(KB): Temporary Interface until Workbox v5. Replace when upgrading.
+interface RequestOptions {
+  event: ExtendableEvent;
+  request: Request | string;
+}
+
 export class AmpDocumentNetworkFirst extends NetworkFirst {
-  _offlineFallbackUrl?: string;
+  private offlineFallbackUrl?: string;
 
   constructor(options: any, offlineFallbackUrl?: string) {
     super(options);
-    this._offlineFallbackUrl = offlineFallbackUrl;
+    this.offlineFallbackUrl = offlineFallbackUrl;
```

Comment:
should we make `offlineFallbackUrl` not nullable then?

Reply:
There is a condition check on line 37. (https://github.com/ampproject/amp-sw/pull/43/files#diff-2e890695ca14f8861f3edd8b215336d8R37)